src/plugins/explore/public/components/query_panel/query_panel_editor/use_query_panel_editor/use_query_panel_editor.ts

```diff
@@ -72,6 +72,10 @@ const languageConfiguration: LanguageConfiguration = {
     { open: "'", close: "'" },
     { open: '`', close: '`' },
   ],
+  comments: {
+    lineComment: '//', // line comment
+    blockComment: ['/*', '*/'], // block comment
+  },
```

Comment:
can you see if there was a test that checks for these other stuff? if there was, might be worth adding this one too so that test fails if someone does something to it by accidently in the future